wp-content/themes/thestanforddaily/template-parts/content.php

```diff
@@ -22,7 +22,7 @@
 		if ( 'post' === get_post_type() ) :
 			?>
 			<div class="entry-meta">
-				<?php tsd_posted_by(); ?> &mdash; <?php tsd_posted_on(); //tsd_comments_count(); ?>
+				<?php tsd_posted_by(); ?> &mdash; <?php tsd_posted_on(); tsd_comments_count(); ?>
```

Comment:
Cool. Eventually we will need to update tsd_comments_count() to actually pull from disqus though. Right now it's kind of useless.

Reply:
Disqus plugin take care of that and it's working correctly right now.